unittests/test_schedulers.py

```diff
@@ -381,11 +392,12 @@ def test_submit(make_job, exec_ctx):
 
     # Additional scheduler-specific checks
     sched_name = minimal_job.scheduler.registered_name
+
     if sched_name == 'local':
         assert [socket.gethostname()] == minimal_job.nodelist
         assert minimal_job.exitcode == 0
         assert minimal_job.state == 'SUCCESS'
-    elif sched_name == ('slurm', 'squeue', 'pbs', 'torque'):
+    elif sched_name in ('slurm', 'squeue', 'pbs', 'torque', 'flux'):
```

Comment:
This might have revealed an actual bug. Is there a way to retrieve the nodes of a job using flux ?

Reply:
@teojgo I think that this has triggered a bug also in other backends, that's why the "CSCS CI" fails. Could you check it?

@vsoch Putting `flux` in that list makes sense only if there is a way to retrieve the nodes where a flux job has run.